From 38fdc7aff2991fd43f0b98fb0bacbc574477449c Mon Sep 17 00:00:00 2001
From: Jeff Layton <jlayton@samba.org>
Date: Sat, 7 Dec 2013 08:54:59 -0500
Subject: [PATCH 1/5] autoconf: fix link of libwbclient

It's currently getting added to $LIBS and being linked into places that
we don't need it.

Signed-off-by: Jeff Layton <jlayton@samba.org>
(cherry picked from commit 2152ccc3d61e5248c70360020a0aa279a24f852c)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---
 aclocal/idmap.m4 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/aclocal/idmap.m4 b/aclocal/idmap.m4
index c5727f6..3ccdae3 100644
--- a/aclocal/idmap.m4
+++ b/aclocal/idmap.m4
@@ -21,6 +21,7 @@ fi
 
 if test $enable_cifsacl != "no" -o $enable_cifsacl != "no"; then
 	ac_wbc_save_LDFLAGS="$LDFLAGS"
+	ac_wbc_save_LIBS="$LIBS"
 	LDFLAGS="$LDFLAGS $WBCLIENT_LIBS"
 	AC_CHECK_LIB(wbclient, wbcSidsToUnixIds, , [
 				if test "$enable_cifsidmap" = "yes"; then
@@ -37,6 +38,7 @@ if test $enable_cifsacl != "no" -o $enable_cifsacl != "no"; then
 				fi
 			])
 	LDFLAGS=$ac_wbc_save_LDFLAGS
+	LIBS=$ac_wbc_save_LIBS
 fi
 
 if test $enable_cifsacl != "no"; then
-- 
2.5.5

